src/main/java/com/eternalcode/parcellockers/gui/ParcelArchiveGUI.java

```diff
@@ -10,12 +10,12 @@ public class ParcelArchiveGUI {
     private static final int[] BORDER_SLOTS = {1, 2, 3, 4, 5, 6, 7, 9, 17, 18, 26, 27, 35, 36, 44, 46, 47, 48, 49, 50, 51, 52};
```

Comment:
mógłbyś to dodać do configu. (jeden komentarz do wszystkich miejsc gdzie robisz taką tablicę)

Reply:
Ciężkie trochę, bordery w GUI nie są zmienne, a liczba faktycznych wyświetlanych pól na paczki jest stała (7 * 4). Reasumując wstawianie tego do configu jest niepożądane